tests/functional/test_validation_check.py

```diff
@@ -136,10 +135,7 @@ def test_validation_check(
             assert_if_fail=False,
         )
 
-        if language == NO_CUSTOM:
-            assert re.search(r"Null value imputation\s+FAILED", stdo)
-        else:
-            assert re.search(r"Null value imputation\s+PASSED", stdo)
+        assert re.search(r"Null value imputation\s+PASSED", stdo)
```

Comment:
The null value imputation test functions properly with scikit-learn models generated by the latest releases.